app/views/content/is-teaching-right-for-me/physics/_article.html.erb

```diff
@@ -0,0 +1,116 @@
+<div class="row inset">
+<section class="col col-720">
+  <p>
+    Teaching physics is an exciting and rewarding career. Schools need more specialist physics teachers, so it's a great choice of subject.
+  </p>
+
+  <p>
+You could inspire your pupils to consider rewarding careers from tackling climate change to artificial intelligence (AI).
+  </p>
+  <p>Tax-free bursaries of £28,000 or scholarships of £30,000 are available for eligible trainee physics teachers.</p>
+
+  </section>
+
+   <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "Spark curiosity in physics",
+      colour: "yellow",
+      image_1_path: "static/images/content/hero-images/0012.jpg",
+      image_1_alt: "Pupils listening to a science lesson.",
+      image_2_path: "static/images/content/hero-images/0007.jpg",
+      image_2_alt: "Teacher talking to a pupil in a classroom.",
+    ) do %>
+       <p>As a physics teacher, you’ll be engaging students in a wide range of experiments that can spark their interest.</p>
+
+  <p>
+  </p>
+    <% end %>
+</section>
+<section class="col col-720">
+      <h2>What you'll be teaching</h2>
+      <p>
+      You'll teach the content of the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-science-programmes-of-study/national-curriculum-in-england-science-programmes-of-study">national curriculum</a>, with opportunities to develop your pupils' ability to think like physicists and see physics as a solution to many challenges. The themes you are likely to cover for 11 to 14 year olds (key stage 3) include:</p>
```

Comment:
Would suggest splitting up this para, with `The themes you are likely to cover for 11 to 14 year olds (key stage 3) include:` on a new line?